tests/test_libinteract.py

```diff
@@ -1,653 +1,1068 @@
-    return np.loadtxt(ref_hb_graph_twochains_file)
+def sb_charged_groups(sb_cg_file):
+    
+    # Charged groups to calculate the electrostatic interactions
+    return li.parse_cgs_file(sb_cg_file)
 
 @pytest.fixture
-def ref_hc_graph_twochains(ref_hc_graph_twochains_file):
-    return np.loadtxt(ref_hc_graph_twochains_file)
+def hb_don_acc(hb_don_acc_file):
+    
+    # Donors and acceptors to calculate the hydrogen bonds
+    return li.parse_hbs_file(hb_don_acc_file)
+
 
-# potential files
 @pytest.fixture
-def ref_potential_file(ref_dir):
-    return "{0}/kb-potential.dat".format(ref_dir)
+def kbp_atomlist(kbp_atoms_file):
+    
+    # Atoms to calculate the statistical potential
+    return li.parse_atomlist(kbp_atoms_file)
+
+
+
+#----------------------------- References ----------------------------#
+ 
+
+
+@pytest.fixture
+def acpsn_references(ref_dirs):
+
+    # Directories containing expected results for the system with
+    # one chain or two chains
+    oc = ref_dirs["one_chain"]
+    tc = ref_dirs["two_chains"]
+
+    # Names of the tables and matrices
+    table = "acpsn"
+    matrix = "acpsn-graph"
+
+    # Suffixes pertaining to the files of the two-chains system
+    # (all contacts, intrachain and intrachain contacts)
+    tc_all = "twochains_all"
+    tc_a = "twochains_intra_A"
+    tc_b = "twochains_intra_B"
+    tc_ab = "twochains_inter_A-B"
+
+    # Expected results for the acPSN
+    return \
+        {"tables" : \
+          {"one_chain" : open(f"{oc}/{table}.csv").readlines(),
+         "two_chains" : \
+           {"all" : open(f"{tc}/{table}_{tc_all}.csv").readlines(),
+            "intra_a" : open(f"{tc}/{table}_{tc_a}.csv").readlines(),
+            "intra_b" : open(f"{tc}/{table}_{tc_b}.csv").readlines(),
+            "inter_ab" : open(f"{tc}/{table}_{tc_ab}.csv").readlines()}},
+         "matrices" : \
+            {"one_chain" : np.loadtxt(f"{oc}/{matrix}.dat"),
+             "two_chains" : \
+               {"all" : np.loadtxt(f"{tc}/{matrix}_{tc_all}.dat"),
+                "intra_a" : np.loadtxt(f"{tc}/{matrix}_{tc_a}.dat"),
+                "intra_b" : np.loadtxt(f"{tc}/{matrix}_{tc_b}.dat"),
+                "inter_ab" : np.loadtxt(f"{tc}/{matrix}_{tc_ab}.dat")}}}
+
+
+@pytest.fixture
+def hc_references(ref_dirs):
+
+    # Directories containing expected results for the system with
+    # one chain or two chains
+    oc = ref_dirs["one_chain"]
+    tc = ref_dirs["two_chains"]
+
+    # Suffix for files pertaining to the rg-corrected hydrophobic
+    # contacts
+    rg = "rg_corrected_all"
+
+    # Names of the tables and matrices
+    table = "hydrophobic-clusters"
+    matrix = "hc-graph"
+
+    # Suffixes pertaining to the files of the two-chains system
+    # (all contacts, intrachain and intrachain contacts)
+    tc_all = "twochains_all"
+    tc_a = "twochains_intra_A"
+    tc_ab = "twochains_inter_A-B"
+
+    # Expected results for the hydrophobic contacts
+    return \
+        {"tables" : \
```

Comment:
we don't need these escapes when using multiple lines to create a nested dictionary

Reply:
Ok! I removed all those that I could find in `test_libinteract.py`.